src/js/utils/dom.js

```diff
@@ -91,3 +91,13 @@ export function stopMonitoringBrowserState() {
     window.removeEventListener('focus', onWindowFocus);
     window.removeEventListener('blur', onWindowBlur);
 }
+
+export function getElementProperties(element) {
+    return {
+        width: element.offsetWidth || 0,
+        height: element.offsetHeight || 0,
+        paddingLeft: window.getComputedStyle(element, null).getPropertyValue('padding-left'),
```

Comment:
Should we cache the result of `getComputedStyle` in a var to avoid calling it 3 times?